01-start/03-begin.html.md

```diff
@@ -565,7 +560,7 @@ As you now have all the tools and knowledge required to be able to create the bl
 
 Congratulations! You now possess all the foundations required to be able to write amazing and powerful web applications like those already in our [Showcase](/docpad/showcase). To recap, you now know how to:
 
-- write documents in any language, markup, pre-processor, templating engine, whatever you wish, by installing the necessary plugin for it and changing the extensions of the document
+- write `render` files in any language, markup, pre-processor, templating engine, whatever you wish, by installing the necessary plugin for it and changing the extensions of the document
```

Comment:
I think you're right that it may be better not to enforce a migration for folks who have something that's working that they're not going to change much. But I recently saw a case where someone accidentally created a directory with the new name when they should have continued to use the old named directory that was already that, and that caused a conflict mysterious to that user.

How about refusing to work with a clear warning when both directories (old name and new name) exist, and asking the user to use just one or the other?